### DISCUSSION TITLE
How can I migrate Milvus data from one machine to another?

### DISCUSSION THREAD

**Discussion by darkerin** [milvus-io/milvus#25294] [Q&A and General discussion]
Both machines are running Ubuntu 20.04 with Milvus 2.2.8 standalone using Docker Compose. I tried directly copying the "volumes" directory to the new machine, but it failed to start. What should I do?  Your help would be greatly appreciated!

**Comment by xiaofan-luan** [comment]
you need to migrate etcd and minio data also.

**Reply by darkerin** [reply]
I used scp to move the entire "volumes" directory to new machine, which includes milvus, etcd, and minio dirs. However, after starting the containers on the new machine, the program runs successfully, but when I use attu to view the collection data, it doesn't display any data in collection.

![image](https://github.com/milvus-io/milvus/assets/136887833/73ca2ef5-ee0e-48c6-a275-95bb0b9f4efb)
